src/media/js/main.js

```diff
@@ -1,25 +1,26 @@
 console.log('Firefox Marketplace Curation Tools');
 
-define('main', [
-    'helpers',  // Must come before mostly everything else.
+define('main', ['init'], function() {
+require([
+    'core/helpers',  // Must come before mostly everything else.
     'helpers_local',
     // 'forms',  // Comment this if your app has no forms.
-    'l10n',
-    'log',
-    'login',  // Comment this if your app does not have accounts.
-    'navigation',
+    'core/l10n',
+    'core/log',
+    'core/login',  // Comment this if your app does not have accounts.
+    'core/navigation',
     'templates',
```

Comment:
`helpers` ends up requiring `nunjucks` to register filters on it. Might not be a bad idea to be explicit though.